subsys/bluetooth/host/l2cap.c

```diff
@@ -90,6 +90,13 @@ static void free_tx_meta_data(struct l2cap_tx_meta_data *data)
 	k_fifo_put(&free_l2cap_tx_meta_data, data);
 }
 
+void bt_l2cap_free_tx_context(void)
+{
+	while (k_fifo_get(&free_l2cap_tx_meta_data, K_NO_WAIT)) {
+
+	}
```

Comment:
Do we not get the same result from simply calling `k_fifo_init(&free_l2cap_tx_meta_data);`?